spec/locales-currencies-tz.html

```diff
@@ -164,59 +164,229 @@ <h1>
+      This section defines how the IANA Time Zone Database should be used by time zone aware implementations.
+      No String may be an available named time zone identifier unless it is a Zone name or a Link name in the IANA Time Zone Database.
+      Available named time zone identifiers returned by ECMAScript built-in objects must use the casing found in the IANA Time Zone Database.
     </p>
+    <p>
+      Each Zone in the IANA Time Zone Database must be a primary time zone identifier and each Link name in the IANA Time Zone Database must be a non-primary time zone identifier that resolves to its corresponding Zone name, with the following exceptions implemented in AvailableNamedTimeZoneIdentifiers:
+    </p>
+    <ul>
+      <li>
+        For historical reasons, *"UTC"* must be a primary time zone identifier.
+        *"Etc/UTC"*, *"Etc/GMT"*, and *"GMT"*, as well as all Link names that resolve to any of them, must be non-primary time identifiers that resolve to "UTC".
+      </li>
+      <li>
+        Any Link name in the TZ column of *zone.tab* of the IANA Time Zone Database must be a primary time zone identifier.
+        For example, both *"Europe/Prague"* and *"Europe/Bratislava"* must be primary time zone identifiers.
+        This requirement guarantees at least one primary time zone identifier for each ISO 3166-2 country code <a href="https://www.iso.org/iso-3166-country-codes.html">https://www.iso.org/iso-3166-country-codes.html</a>.
+        Also, future changes to time zone rules of one country will not affect ECMAScript programs that use another country's time zones, unless those countries' territorial boundaries have also changed.
+      </li>
+      <li>
+        Any Link name that is not listed in the TZ column of *zone.tab* and that represents a geographical area entirely contained within the territory of a single ISO 3166-2 country code <a href="https://www.iso.org/iso-3166-country-codes.html">https://www.iso.org/iso-3166-country-codes.html</a> must resolve to a primary identifier that also represents a geographical area entirely contained within the territory of the same ISO 3166-2 country code.
+        For example, *"Atlantic/Jan_Mayen"* must resolve to *"Arctic/Longyearbyen"*.
+      </li>
+      <li>
+        <emu-not-ref>Legacy</emu-not-ref> POSIX Zone names must be non-primary time zone identifiers that resolve to their closest non-legacy equivalents, as shown in the table below:
+      </li>
+    </ul>
+
+    <emu-table id="table-posix-time-zone-identifier-mapping">
+      <emu-caption><emu-not-ref>Legacy</emu-not-ref> POSIX Time Zone Identifier Mapping</emu-caption>
+      <table class="real-table">
+        <thead style="white-space: nowrap">
+          <tr>
+            <th><emu-not-ref>Legacy</emu-not-ref> POSIX Zone Name</th>
+            <th>Primary Time Zone Identifier</th>
+          </tr>
+        </thead>
```

Comment:
I don't think this use of "POSIX" is correct; https://data.iana.org/time-zones/theory.html#naming just refers to "legacy names".
```suggestion
        "<emu-not-ref>Legacy names</emu-not-ref>" must be non-primary time zone identifiers that resolve to their closest non-legacy equivalents, as shown in <emu-xref href="#table-legacy-time-zone-identifier-mapping"></emu-xref> below:
      </li>
    </ul>

    <emu-table id="table-legacy-time-zone-identifier-mapping">
      <emu-caption><emu-not-ref>Legacy</emu-not-ref> Time Zone Identifier Mapping</emu-caption>
      <table class="real-table">
        <thead style="white-space: nowrap">
          <tr>
            <th><emu-not-ref>Legacy Name</emu-not-ref></th>
            <th>Primary Time Zone Identifier</th>
          </tr>
        </thead>
```